lib/plugins/aws/deploy/lib/monitorStack.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const async = require('async');
+const chalk = require('chalk');
+
+module.exports = {
+  monitorStack(action, cfData, frequency) {
+    // Skip monitoring if a deployment should not be performed
+    if (this.options.noDeploy) return BbPromise.bind(this).then(BbPromise.resolve());
+
+    // Skip monitoring if stack was already created
+    if (typeof cfData === 'undefined') return BbPromise.bind(this).then(BbPromise.resolve());
```

Comment:
Does this check for `undefined` really means that it's not created or might it be possible that some kind of error occurred here?